Dockerfile

```diff
@@ -25,11 +25,7 @@ ENV BUNDLE_APP_CONFIG=".bundle_app_config"
 
 # Update rubygems
 ARG RUBYGEMS_VERSION
-RUN gem update --system ${RUBYGEMS_VERSION} --no-document && \
-  # rubygems-update is completely unused after the `gem update --system` process
-  gem uninstall rubygems-update -x && \
```

Comment:
Interesting! I think the comment above is still true, and I would expect `gem uninstall rubygems-update -x` to still work. I was actually considering recently that this should be the default behavior and no `ruby gems-update` gem should be left around. Maybe some rubygems regression?